src/Device/RecordedFlight.hpp

```diff
@@ -49,6 +49,14 @@ struct RecordedFlightInfo : FlightInfo {
      */
     unsigned flytec;
 
+    struct {
+      /**
+       * Flight id and igc file size, used by the LX Eos driver.
+       */
```

Comment:
A doxygen comment documents only the following field, not both. Split this comment.